synadm/cli/user.py

```diff
@@ -358,10 +358,18 @@ def name_extra(self):
     removes their active access tokens, resets their password, kicks them out
     of all rooms and deletes third-party identifiers (to prevent the user
     requesting a password reset). See also "user deactivate" command.""")
+@optgroup.option(
+    "--user-type", "user_type", type=str,
+    help="""Change the type of the user. Currently allowed values are 'bot'
+    and support.""")
```

Comment:
Inconsistent use of single quotes, here's what I suggest:
```suggestion
    and 'support'.""")
```